### ISSUE TITLE
Build option for VIRTIO_LEGACY

### ISSUE THREAD

**Issue by milisarge** [nuta/resea#24]
i run the system in virtio legacy with `export VIRTIO_LEGACY=y` command. Should we add a build option for this?

**Comment by nuta** [comment]
You mean you want to edit QEMU options (e.g. `$VIRTIO_LEGACY`, `$SMP`, `$GUI`) in `make menuconfig`?

**Comment by milisarge** [comment]
yes just i think to it should be in menuconfig or?  for ease of building

**Comment by nuta** [comment]
Sounds good to me. I added this feature to the backlog. Thank you!
